acceptance/testdata/analyzer/analyze-image/Dockerfile

```diff
@@ -9,3 +9,7 @@ WORKDIR /layers
 ENV CNB_USER_ID=2222
 
 ENV CNB_GROUP_ID=3333
+
+RUN chmod -R 777 /some-dir
+
+RUN chmod -R 777 /layers
```

Comment:
This line is in order to make the test on line 647 work.
I'm not sure if it's ok to add it, although we're actually doing something similar as part of the `Privileges` function of the `analyzer`.

Reply:
I agree that it's not a great idea to run `chmod -R` on both directories.
I'm not sure that I fully understand what you wrote. In the commit I'll push in 2 minutes, I've replaced `RUN chmod -R` in the dockerfile to `RUN chown -R $CNB_USER_ID:$CNB_GROUP_ID` and everything works.
Do you want me to make any other changes?

Reply:
I was suggesting that we change the default user from `root` and only run explicitly as `root` when testing behavior that requires it (e.g. connecting to the daemon socket, chowning the layers dir). But as you point out that isn't necessary to test this feature and I think we can make that change at a later point if we desire.